lib/geokit/multi_geocoder.rb

```diff
@@ -45,7 +45,7 @@ def self.do_geocode(address, *args)
       # This method will call one or more geocoders in the order specified in
       # the configuration until one of the geocoders work, only this time it's
       # going to try to reverse geocode a geographical point.
-      def self.do_reverse_geocode(latlng)
+      def self.do_reverse_geocode(latlng, *args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used.